examples/application/evaluate_gaming_traffic.cc

```diff
@@ -118,32 +135,44 @@ main (int argc, char *argv[])
 
   Ipv4InterfaceContainer interfaces = address.Assign (devices);
 
-  GamingStreamingServerHelper gamingStreamingHelper (game, interfaces.GetAddress (0), 9);
-  gamingStreamingHelper.SetAttribute("BitRate", DoubleValue(bitrate));
```

Comment:
Space after `SetAttribute` and `DoubleValue`.

Reply:
@salmanmohebi you could actually run a style check on all your new classes so these things are automatically corrected

Reply:
> @salmanmohebi you could actually run a style check on all your new classes so these things are automatically corrected

I usually do. I will check all my classes again now.